### PR TITLE
Adjust README.md so that CMake easily use vcpkg-installed packages

### PR DESCRIPTION
**Pull request by ret2libc**
Provide instructions similar to those in https://vcpkg.io/en/getting-started.html .

### CODE REVIEW

**Comment by ret2libc** [comment]
I wasn't actually able to install things with `vcpkg install pe-parse`. I had to run `vcpkg install`, but even then CMake was not actually able to discover the libraries needed.

By using `CMAKE_TOOLCHAIN_FILE=<vcpkg-path>/scripts/buildsystems/vcpkg.cmake` cmake automatically install the deps and adds the right paths to use those libs.

File: README.md [hunk 1]
```diff
@@ -83,9 +73,9 @@ The `svcli` utility also provides a small example of using *μthenticode*'s APIs
 You can build it by passing `-DBUILD_SVCLI=1` to `cmake`:
 
 ```bash
-cmake -DBUILD_SVCLI=1 ..
-cmake --build .
-./src/svcli/svcli /path/to/some.exe
+cmake -DBUILD_SVCLI=1 -B build -S . -DCMAKE_TOOLCHAIN_FILE=<vcpkg-path>/scripts/buildsystems/vcpkg.cmake
+cmake --build build
+./build/bin/svcli /path/to/some.exe
```
Comment (woodruffw) [inline README.md:78]: It's a little confusing, but I _think_ this will be:

```suggestion
./build/src/svcli /path/to/some.exe
```

Comment (ret2libc) [reply README.md:78]: Ah right, sorry I saw `svcli` in the `bin` directory because I also installed it (which created the `bin` directory)

**Comment by woodruffw** [comment]
Thanks for this!

Yeah, the steps before were very implicit: they assumed that the user had `vcpkg` configured environmentally, which isn't how `vcpkg`'s own docs encourage things. This is much better!

**Comment by woodruffw** [comment]
Judiciously ignoring the CLA for an employee.

**Pull request merged**